CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
+- `GetActorSpatialPosition` now returns the last spectator sync location while the player is spectating.
 - The default cloud launch configuration is now empty.
-- Fixed an crash caused by attempting to read schema from an unloaded class.
-- Unresolved object references in replicated arrays of structs should now be properly handled and eventually resolved.
-- Fix tombstone-related assert that could fire and bring down the editor.
-- Actors placed in the level with bNetLoadOnClient=false that go out of view will now be reloaded if they come back into view.
-- Fix crash in SpatialDebugger caused by dereference of invalid weak pointer.
-- Fixed connection error when using spatial cloud connect external.
-- The command line argument "receptionistHost <URL>" will now not override connections to "127.0.0.1".
-- The receptionist will now be used for appropriate URLs after connecting to a locator URL.
+- We've fixed a crash that happened when the GDK attempted to read schema from an unloaded class.
+- We now properly handle (and eventually resolve) unresolved object references in replicated arrays of structs.
+- We've fixed a tombstone-related assert that could fire and bring down the editor.
+- If an Actor that is placed in the level with `bNetLoadOnClient=false` goes out of view, it is now reloaded if it comes back into view.
+- We've fixed a crash in `SpatialDebugger` that was caused by the dereference of an invalid weak pointer.
+- We've fixed a connection error that occurred when using `spatial cloud connect external`.
+- The command line argument `receptionistHost <URL>` no longer overrides connections to `127.0.0.1`.
+- The Receptionist will now be used for appropriate URLs after connecting to a locator URL.
 - You can now access the worker flags via `USpatialStatics::GetWorkerFlag` instead of `USpatialWorkerFlags::GetWorkerFlag`.
-- Fix crash in SpatialDebugger when GDK-space load balancing is disabled.
-- Fixed issue where schema database failed to load previous saved state when working in editor.
-- Attempting to launch a cloud deployment will now run the spatial auth process as it is required. Previously the deployment would simply fail.
-- Minor spelling fix to connection log message.
+- We've fixed a crash in `SpatialDebugger` when GDK-space load balancing is disabled.
+- The schema database no longer fails to load previous saved state when working in the Unreal Editor.
+- If you attempt to launch a cloud deployment, this now runs the `spatial auth` process as required. Previously the deployment would simply fail.
+- We've made a minor spelling fix to the connection log message.
 - Added %s token to debug strings in GlobalStateManager to display actor class name in log.
-- The server no longer crashes, when received RPCs are processed recursively.
+- The server no longer crashes when received RPCs are processed recursively.
 - Fix to serialize SoftObjectPointers when they are not resolved yet.
```

Comment:
I don’t know what this release note (or the two below) are trying to convey. E.g. does this one mean that we now serialize SoftObjectPointers when they are not yet resolved (whereas we didn’t before)? Or that we’ve done this for a while, but we’ve just made a fix that relates to how we do this?